projects/packages/waf/src/class-waf-runner.php

```diff
@@ -257,8 +257,9 @@ public static function run() {
 
 		// remove the custom error handler, so we don't interfere with the site.
 		restore_error_handler();
-		// phpcs:ignore
-		ini_set( 'display_errors', $display_errors );
+
+		// Restore the original value.
+		ini_set( 'display_errors', $display_errors ); // phpcs:ignore
```

Comment:
It'd be nice to be more specific with the `ignore` here, and add a comment as to why this gets ignored.

Reply:
Did that in 86c92f6293d0c6157291393053f23fb68a9b460a